code/game/machinery/slotmachine.dm

```diff
@@ -10,7 +10,7 @@
 #define JACKPOT 10000
 #define SPIN_TIME 65 //As always, deciseconds.
 #define REEL_DEACTIVATE_DELAY 7
-#define SEVEN "<font color='red'>7</font>"
+#define SEVEN "fa-7"
```

Comment:
```suggestion
#define JACKPOT_SEVENS FA_ICON_7
```
This will let you use ``JACKPOT_SEVENS`` in this file while still being in reference to the icon, so it is future proofed while not being bad to look at in the file.